mysite/bugsets/templates/edit_index.html

```diff
@@ -34,6 +34,18 @@
     <form action="{{ request.get_full_path }}" method="post">
       {% csrf_token %}
       {{ form.as_p }}
+      <script>
```

Comment:
Yes. Please see the linked file in [my comment on the issue](https://github.com/openhatch/oh-mainline/issues/1385#issuecomment-92408871) for where to put this code.